content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,157 @@
+
+* A [S3](https://aws.amazon.com/s3/) bucket: `s3://<S3 Bucket Name>[/<Optional Prefix>]`.
+* A [SQS](https://aws.amazon.com/sqs/) queue that passes along [SNS notifications](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/configure-sns-notifications-for-cloudtrail.html) about new log files: `sqs://<SQS Queue Name>`
+* A local filesystem path: `Any path`.
+
+The plugin also exports fields that extract information from a cloudtrail event, such as the event time, the aws region, S3 bucket/EC2 instance name, etc.
+
+The plugin uses the same authentication mechanisms used by the [AWS Go SDK](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials):
+
+* **Environment Variables**: specify the aws region with `AWS_REGION=xxx`, the access key id with `AWS_ACCESS_KEY_ID=xxx`, and the secret key with `AWS_SECRET_ACCESS_KEY=xxx`.
+* **Shared Configuration Files**: specify the aws region in a file at `$HOME/.aws/config` and the credentials in a file at `$HOME/.aws/credentials`.
+
+#### New Rules for AWS Cloudtrail
+
+Along with the plugin, we've created new Falco rules that look for suspicious/notable/interesting activity in cloudtrail logs. Here is an example rule "Console Login Without MFA" (Multi Factor Authentication):
+
+```yaml
+- rule: Console Login Without MFA
+  desc: Detect a console login without MFA.
+  condition:
+    ct.name="ConsoleLogin" and ct.error=""
+    and json.value[/userIdentity/type]!="AssumedRole" and json.value[/responseElements/ConsoleLogin]="Success"
+    and json.value[/additionalEventData/MFAUsed]="No"
+  output:
+    Detected a console login without MFA (requesting user=%ct.user, requesting IP=%ct.srcip, AWS region=%ct.region)
+  priority: CRITICAL
+  source: aws_cloudtrail
+```
+
+The rule source is `aws_cloudtrail`. When falco is run with a loaded cloudtrail plugin, it will evaluate these rules because the plugin's event source matches the source in the rule. The fields `ct.name`, `ct.error`, `ct.srcip`, etc are all resolved by the plugin via a call to the plugin's extract fields function. For example, when `ct.name` is "ConsoleLogin", that represents a console login attempt. The full set of fields supported by the cloudtrail plugin are in the [README](https://github.com/falcosecurity/plugins/blob/master/plugins/cloudtrail/README.md).
+
+It also happens to be the case that the underlying event payload is JSON, with [this format](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference.html). This allows the json plugin to extract additional values from the event, via `json.value` plus some JSON pointer argument. For example, when `json.value[/responseElements/ConsoleLogin]` is "Success", that represents a successful login attempt, and when `json.value[/additionalEventData/MFAUsed]` is "No", the event represents a console login where MFA was not used.
```

Comment:
It is also applicable if the underlying event payload is JSON with [this format](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference.html). This allows the JSON plugin to extract additional values from the event, via `json.value` plus some JSON pointer argument. For example, matching `json.value[/responseElements/ConsoleLogin]` with "Success" represents a successful login attempt, and  `json.value[/additionalEventData/MFAUsed]` with "No" represents a console login where MFA was not used.